numpydoc/docscrape.py

```diff
@@ -327,6 +327,11 @@ def _parse(self):
             if not section.startswith('..'):
                 section = (s.capitalize() for s in section.split(' '))
                 section = ' '.join(section)
+                if self[section]:
```

Comment:
Why would that be better? If `section` is not a key of the dictionary (nor starts with `..`), an error will be raise later on (l.344). But maybe I am missing something.